clusters/hive/hive-alerts_prometheusrule.yaml

```diff
@@ -23,3 +23,19 @@ spec:
       for: 5m
       labels:
         severity: critical
+  - name: hive-abnormal
+    rules:
+    - alert: hive-blocked-deprovision
+      annotations:
+        message: The deprovisioning of the cluster deployment {{ $labels.exported_namespace }}/{{ $labels.cluster_deployment }} has not been completed over 24 hours. Please check the pod log in the namespace.
+        summary: The deprovisioning of a cluster takes too long.
+        description: Long deprovisioning of a cluster usually caused by some resources on the cloud that cannot be deleted by the OpenShift installer.
+          It still occupies the quota of the cloud account. The owners of the cloud accounts should be informed to handle the case manually.
+          Mostly it can be resolved by a fix to a bug of the OpenShift installer or a support ticket to the cloud platform.
+          In some/rare cases, a test could abuse the cloud credentials and create resources on the cloud and forget/fail to clean them up.
+        runbook_url: https://github.com/openshift/release/blob/master/docs/dptp-triage-sop/hive.md#hive-blocked-deprovision
+      expr: |
+        hive_cluster_deployment_deprovision_underway_seconds{exported_namespace!~"ci-ocp-4-10-amd64-aws-us-east-1-jwp27|ci-ocp-4-10-amd64-aws-us-east-1-x6cr9|ci-ocp-4-10-amd64-aws-us-east-1-x6jdp|ci-ocp-4-10-amd64-aws-us-east-1-jfrb6|ci-ocp-4-10-amd64-aws-us-east-1-l2cg5|hypershift-ocp-4-9-amd64-aws-us-east-1-wns2n"} > 60*60*24
```

Comment:
See https://issues.redhat.com/browse/DPTP-2897?focusedCommentId=21818191&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-21818191 about why those namespaces are ignored.

We can remove them when the cases are resolved.